### PR TITLE
React accordion component

### PR DESCRIPTION
**Pull request by KiraHike**
image

### CODE REVIEW

File: react-accordion-component/src/accordian.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      {
+        id: 3,
+        name: 'JavaScript',
+        about: `JavaScript, often abbreviated as JS, is a high-level, interpreted
+              programming language that conforms to the ECMAScript specification.
+              JavaScript has curly-bracket syntax, dynamic typing, prototype-based
+              object-orientation and first-class functions.`
+      }
+    ];
+  }
```
Comment (thebearingedge) [inline react-accordion-component/src/accordian.jsx:34]: These should be passed into the component via props (in `index.jsx`).

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Great, just one small change.

**Comment by KiraHike** [comment]
revised

**Review by thebearingedge — APPROVED** [review]
👏

**Pull request merged**